notebooks/EPM_demo.ipynb

```diff
@@ -174,9 +174,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "amadeusgpt-minimal",
+   "display_name": "amadeusgpt-cpu",
    "language": "python",
-   "name": "python3"
+   "name": "amadeusgpt-cpu"
```

Comment:
drop -cpu

Reply:
This is just a meta info that stores my conda environment name. I don't know why it's there as I hit clear all outputs